fa20/syllabus.yml

```diff
@@ -2,17 +2,18 @@
   required:
     id: 8f863b4f2ad5d2600cb549cf885be1065f9423efbd7526a3f3e73457fab26bb7
     date: !Timestamp 2020-09-15 18:00:00
-    title: meeting00
+    title: a-few-useful-things
```

Comment:
Didn't clarify this, whoops. 😅  This should by stylized as a proper title, so in this case it would be... "A Few Useful Things."